pkg/model/loader.go

```diff
@@ -128,6 +128,17 @@ func (ml *ModelLoader) LoadModel(modelName string, loader func(string, string) (
 	return model, nil
 }
 
+func (ml *ModelLoader) UnloadModel(modelName string) error {
+	ml.mu.Lock()
+	defer ml.mu.Unlock()
+	if _, ok := ml.models[modelName]; !ok {
+		return fmt.Errorf("model %s not found", modelName)
+	}
+
+	delete(ml.models, modelName)
```

Comment:
this is a repetition, its already handled here: https://github.com/go-skynet/LocalAI/blob/f143a32e5b14f980c65f1f4f0cbb200b82bb6412/pkg/model/process.go#L36